testdata/project-v4-declarative-v1/monitoring/metrics/util/util.go

```diff
@@ -0,0 +1,145 @@
+package util
```

Comment:
Should not this kind of implementation fit better in a lib such as controller-runtime or operator-lib or a new one since prometheus-lib?

If all projects would need to use this func so that does not seems reasonable to be scaffolded in a plugin for the final projects. That really seems to be more appropriated in a lib (ihmo)

Reply:
@camilamacedo86 Not sure I understand right, but this implementation (util) is only for adding Prometheus metrics  to the operator.

Reply:
I think what @camilamacedo86 is trying to point out here is that if every operator that wants to use metrics will have to use these utility functions, it may be better to make them a Go library that can be imported rather than scaffolded in every project.

I guess this ends up posing the question - Will every operator that wants to use metrics need to implement these utility functions?